ckan_cloud_operator/providers/ckan/instance/manager.py

```diff
@@ -70,6 +82,19 @@ def update(instance_id_or_name, override_spec=None, persist_overrides=False, wai
     else:
         pre_update_hook_data = deployment_manager.pre_update_hook(instance_id, instance_type, instance, override_spec,
                                                                   skip_route)
+
+        if config_manager.get('use-cloud-native-storage', secret_name=CONFIG_NAME):
```

Comment:
why not test for `use_cloud_storage here`?

Reply:
Done